app/controllers/admin/deals_controller.rb

```diff
@@ -0,0 +1,66 @@
+module Admin
+  class DealsController < AdminController
+    before_action :set_deal, only: [:show, :edit, :update, :destroy]
+
+    def index
+      @deals = Deal.with_attached_images.page(params[:page]).per(1)
```

Comment:
Here page number 1 is fixed, so everytime first page will be displayed. use 1 if page number is not presentin the params